src/UI/Http/Rest/Controller/CommandQueryController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UI\Http\Rest\Controller;
+
+use App\UI\Http\Rest\Response\JsonApiFormatter;
+use League\Tactician\CommandBus;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class CommandQueryController extends QueryController
```

Comment:
Time to move QueryContoller and CommandController to traits?

Reply:
Do you think a trait would be better than an abstract class? The only problem I see its traits can't have constants if I'm not mistaken.

Reply:
I've to rethink this